.github/workflows/integration_tests.yml

```diff
@@ -61,4 +60,5 @@ jobs:
           AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
           # Todo : retrieve this layer version automatically from AWS
           NODE_LAYER_VERSION: 55
+          PYTHON_LAYER_VERSION: 41
```

Comment:
Do we still have a plan to automatically retrieve this from AWS?

Reply:
✅